_posts/en/0500-10-24-tm-admin_en.md

```diff
@@ -123,9 +126,9 @@ Learn more about how to add descriptions and instructions to the project and get
 
 You can be much more precise to draw the area of interest with a geospatial desktop application like JOSM or QGIS, than with the slippy web map inside the Tasking Manager. And you can also determine special task shapes and sizes beforehand.
```

Comment:
Suggestion: 
Geospatial desktop applications like JOSM or QGIS allow mappers to more precisely draw the areas of interest, compared to the slippy web map inside the Tasking Manager. You can also determine special task shapes and sizes beforehand.